### DISCUSSION TITLE
Directions / Route

### DISCUSSION THREAD

**Discussion by dougg0k** [react-native-maps/react-native-maps#5187] [Feature requests]
Hi,

Is there a way to set directions path from start to end?

Like you see in this library https://github.com/bramus/react-native-maps-directions

If not, could this be added? That lib doesnt seem to be maintained for some time.

**Comment by salah-ghanim** [comment]
@dougg0k you really don't need a library for that per say, it's simply creating a map polygon out of the google directions API response. 

but we could think of this feature request if more want it, since it's rather simple to implement

**Comment by dougg0k** [comment]
That feature would be nice, with a start / end (lat/lng).

Ideally support on other maps too like openstreetmap, besides gmaps.
